server/views/components/main-media/index.njk

```diff
@@ -1,15 +1,30 @@
-{% extends 'components/captioned-image/index.njk' %}
-{% block figcaption %}
-  {% component 'wobbly-edge', {background: 'cream'} %}
-  {% if model.caption %}
-    <div class="row row--cream">
-      <div class="container">
-        <div class="grid">
-          <div class="{{ {m:6, xl:10, shiftM:1, shiftXl:1} | gridClasses }}">
-            {{ super() }}
+{% if model.type === 'picture' %}
+  {% extends 'components/captioned-image/index.njk' %}
+  {% block figcaption %}
+    {% component 'wobbly-edge', {background: 'cream'} %}
+    {% if model.caption %}
+      <div class="row row--cream">
+        <div class="container">
+          <div class="grid">
+            <div class="{{ {m:6, xl:10, shiftM:1, shiftXl:1} | gridClasses }}">
+              {{ super() }}
+            </div>
           </div>
         </div>
       </div>
+    {% endif %}
+  {% endblock %}
+{% elif model.type === 'video' %}
+  <div class="row row--dark-black {{ {s:4, m:8, l:10} | spacingClasses({padding: ['top', 'bottom']}) }}">
+    <div class="container">
+      <div class="grid">
+        <div class="{{ {s:4, m:8, l:8, shiftL:2} | gridClasses }}">
+          {% componentV2 'iframed-video', {
+            embedUrl: model.embedUrl
+          } %}
+        </div>
+      </div>
     </div>
-  {% endif %}
-{% endblock %}
+  </div>
+  {% component 'wobbly-edge', {background: 'cream'} %}
```

Comment:
We have a wobbly edge on the video?
Isn't that going to be distracting while watching?